data/json/items/vehicle/solar.json

```diff
@@ -16,6 +16,15 @@
     "price": 90000,
     "price_postapoc": 1000
   },
+  {
+    "type": "GENERIC",
+    "copy-from": "solar_panel",
```

Comment:
Are current foldable solar panels able to charge more than some personal electronics? I see the price of twelve hundred but I think most foldable solar panels are a tenth of that price but provide only a trickle of power.

Reply:
I based the foldable panel prices on the reinforced ones, while the volume and charge rate are loosely based on solar backpacks. What prices would you recommend?

Reply:
I guess more what I'm trying to say is what the power generated by the vehicle panels?  The largest commercially available folding solar panel I could find with a quick google search is 500 watt.   http://energywave.net/product/500-watt-foldable-solar-panel/  I'm trying to understand if that prevents a foldable vehicle lever solar panel from being feasible.  So basically I am thinking that barring evidence otherwise foldable solar panels need to have a power generation of 500 watts max.

Reply:
Oh well in that case this is well under that limit. According to Kevin one epower is 1 joule, and a watt is 1 joule per second. These have 20 epower which, assuming it generates that for every turn (1 second each) translates into 20 watts.